crates/astria-telemetry/src/metrics/builders.rs

```diff
@@ -115,6 +113,16 @@ impl ConfigBuilder {
                 .map_err(|error| Error::StartListening(error.into()))?
         } else {
             let recorder = bucket_builder.builder.build_recorder();
+
+            let recorder_handle = recorder.handle();
+            tokio::spawn(async move {
+                let mut interval = tokio::time::interval(std::time::Duration::from_secs(5));
```

Comment:
This likely is not an issue, but let's set this to to `MissedTickBehavior::Skip` to be conservative: https://docs.rs/tokio/1.45.1/tokio/time/struct.Interval.html#method.set_missed_tick_behavior